### PR TITLE
LEAP-Pangeo paper blog post stub

### PR DESCRIPTION
**Pull request by jnywong**
Related to https://github.com/2i2c-org/meta/issues/1185

### CODE REVIEW

**Comment by jnywong** [comment]
Hey @jbusecke!

I've just published a minimal blog post stub on your recent work. Let me know if there are any more cross-links I can add to it if you would like to expand on it in future 👍

File: content/blog/2024/leap-pangeo-paper/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Researchers at LEAP-Pangeo investigate overlooked sub-grid air-sea heat flux in climate models
+date: "2024-06-19"
```
Comment (jbusecke) [inline content/blog/2024/leap-pangeo-paper/index.md:3]: The [post](https://2i2c.org/blog/2024/leap-pangeo-paper/) somehow shows May as publish date? Not sure how that happens, but just noticed it.

**Comment by jbusecke** [comment]
Maybe for some easy digestable outline you could add a link to my twitter post? https://x.com/JuliusBusecke/status/1792930908900630735

**Comment by jbusecke** [comment]
Otherwise this is great. Thanks for the work.

File: content/blog/2024/leap-pangeo-paper/index.md (hunk 1 above)
Comment (jnywong) [reply content/blog/2024/leap-pangeo-paper/index.md:3]: Ah yes we are "backdating" some posts as we are writing up quite a lot of items at once rn 😅

**Pull request merged**